sql/core/src/test/resources/sql-tests/results/postgreSQL/date.sql.out

```diff
@@ -584,7 +584,7 @@ select make_date(-44, 3, 15)
 -- !query schema
 struct<make_date(-44, 3, 15):date>
 -- !query output
--0044-03-15
+0045-03-15
```

Comment:
This is compatible with Spark 2.4 because it uses SimpleDateFormat which cannot format negative years.